api/app/controllers/review.controller.ts

```diff
@@ -222,3 +226,74 @@ export async function submit(req: Request, res: Response) {
     });
   }
 }
+
+export const getAllReviewsForAdmin = async (req: Request, res: Response) => {
+  const id = Number(req.query.id);
+  const submitted_at = req.query.submitted_at as string | undefined;
```

Comment:
lets follow the camelCase convention for query params that's throughout the rest controllers. applies for all other query params
```suggestion
  const submittedAt = req.query.submitted_at as string | undefined;
```